spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::SpecificMatcher do
+  it 'does not register an offense for abstract matcher when elements with' \
+    'no matcher specified in detail' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_selector('article')
+      expect(page).to have_none_of_selectors('body')
+      expect(page).to have_css('tbody')
+    RUBY
+  end
+
+  it 'does not register an offense for abstract matcher when elements with' \
+    'not element' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_no_css('.a')
+      expect(page).to has_css?('#button')
+      expect(page).to has_no_css?('[table]')
```

Comment:
Is this a correct usage of `has_css?`?
In docs, I see this method used as a node matcher, not as an RSpec predicate matcher:
```ruby
expect(page.has_css?('p#foo')).to be true
```
vs
```
expect(page).to have_css('p#foo')
```

I would rather prefer only checking the RSpec matcher, and let something like `rspec-capybara` to take care of `has_css?`.

Reply:
> I would rather prefer only checking the RSpec matcher, and let something like rspec-capybara to take care of has_css?.

It certainly looks better that way.